docs/tutorial/tutorial-5/android.rst

```diff
@@ -2,9 +2,159 @@
+      javac 1.8.0_242
+
+Create an Android app and compile it
+====================================
+
+The process of deploying an application to Android is very similar to the
+process for deploying as a desktop application. First, you run the ``create``
+command. This downloads an Android app template and adds your Python code to it.
+
+.. code-block:: bash
+
+  (beeware-venv) $ briefcase create android
+
+  [helloworld] Generating application template...
+  Using app template: https://github.com/beeware/briefcase-android-gradle-template.git
+  ...
+  [helloworld] Installing support package...
+  ...
+  [helloworld] Installing dependencies...
+  ...
+  [helloworld] Installing application code...
+  ...
+  [helloworld] Installing application resources...
+  ...
+  [helloworld] Application created.
+
+Once this completes, we'll now have an ``android`` directory in your project.
+This directory will contain a ``Hello World`` folder, which will contain an
+Android project configured to use gradle, your application code, and a support
+package containing the Python interpreter and the `rubicon-java` library.
+
+In the process, Briefcase downloads the Android SDK. You may be prompted to
+accept Google's Android SDK license.
+
+You can then use Briefcase's ``build`` command to compile this into an Android
+APK app file. It will take more than two minutes the first time, depending on
+your Internet connection speed, and then should take less than 30 seconds every
+successive time.
+
+.. code-block:: bash
+
+  (beeware-venv) $ briefcase build android
+  [helloworld] Building Android APK...
+  Starting a Gradle Daemon
+  ...
+  BUILD SUCCESSFUL in 1m 1s
+  28 actionable tasks: 17 executed, 11 up-to-date
+  [helloworld] Built android/Hello World/app/build/outputs/apk/debug/app-debug.apk
+
+Run the app
+===========
+
+We're now ready to run our application. You can use Briefcase's ``run`` command
+to run the app on an Android device, either physical or virtual. On Android,
+the ``run`` command requires that you specify which device to run on.
+
+The easiest way to configure an Android device is to use Briefcase's built-in
+recommendations for how to set up an Android device. Run this command:
+
+.. code-block:: bash
+
+  (beeware-venv) $ briefcase run android
+
+  No Android device was specified. Please specify a specific device on which
+  to run the app by passing `-d <device_id>`.
+
+  ...
+
+  If you do not see any devices, you can create and start an emulator by running:
+
+  ...
+
+
+Examine the specific output on your system, and run the commands provided
+until you see an Android virtual device on your screen. Once an Android virtual
+device is running, you can launch the app with Briefcase by running this command:
+
+.. code-block:: bash
+
+  (beeware-venv) $ briefcase run android -d emulator-5554
+
+At this point, you should see the app launch on your virtual device.
+
+If you have a physical Android device you would prefer to run the app on,
```

Comment:
There are some other steps here - we should document how to enable debug mode, and how to diagnose if you have already enabled debug mode.

Reply:
It's also worth adding "run on a physical device" as a separate section in this tutorial, rather than an "if you'd prefer" footnote, with an "if you don't have a physical Android device, you can skip this section" note.

Reply:
I've added a lightweight version of this, and linked to Google's docs. Let me know what you think.